language-design.md

```diff
@@ -106,7 +129,9 @@ you do so? If not, why not?
 
 **Response**
 
-
+I think there is a lot of truth in what Cook is saying. The closer you get to natural language, the more of an expectation you have to use it _like_ natural language. I think this is problematic because it creates cognitive disonnance from programmers, as their experience in programming (which, at this point, is nothing like talking) is too similar to their experience in using natural language every day. For this reason, it seems to me that on eshould be extremely careful in how much natural language is included in the design of a language. Personally, I would want to avoid associations with phrases and really common words, things like "do thing x times" or something like that. Instead, I would want to use words like "loop" or "do while," which are not as common in our everyday and as thus can form their own identities as words within a programming context.
+In designing a DSL, I would be very careful using natural language. I think I wouldn't go further than using a certain word that evokes a certain feeling or purpose, or can very simply be connected to the task that it completes in the concext of that language. I would be hesitant to create false expectations or cognitive dissonance among the users of my DSL. There is a testing language that works with Ruby called Cucumber, which uses a lot of natural language in the asserting and testing of features; For me, it was extremely frustrating to use Cucumber because it would error at the smallest of deviations from the syntax, despite feeling as though it should work fine if it isn't exactly in the correct syntax. I think this frustration stems from the fact that Cucumber made me feel like I was talking in English to my computer, when in truth, I was simply using syntax that felt like "syntax" I use every day.
+All that said, I think that intuitive and/or verbose method names are good programming practice, because they help the programmer contextualize the intentions of specific programs or functions. However, the language itself shouldn't be projecting its expectations of natural language onto the programmer, when in truth it expects a particular syntax in order to work correctly. 
 
```

Comment:
I like what you said about using less common words/phrases like "loop"/"do while" in order do disassociate programming and natural language.  In a way it makes a lot of sense, but in another way it doesn't.  --> What's the point of incorporating natural language in programming if we're going to end up trying to disassociate ourselves from it?  If we're trying to use words that _don't_ sound like everyday words, why not use random, shorter "words" (eg, "yzu" instead of "extends")?  They would have even less connection to everyday words we use.